vm/systemSmartContracts/delegation_test.go

```diff
@@ -285,12 +288,15 @@ func TestDelegationSystemSC_ExecuteNilArgsShouldErr(t *testing.T) {
 	t.Parallel()
 
 	args := createMockArgumentsForDelegation()
-	eei, _ := NewVMContext(
-		&mock.BlockChainHookStub{},
-		hooks.NewVMCryptoHook(),
-		&mock.ArgumentParserMock{},
-		&stateMock.AccountsStub{},
-		&mock.RaterMock{})
+	eei, _ := NewVMContext(VMContextArgs{
+		BlockChainHook:                          &mock.BlockChainHookStub{},
+		CryptoHook:                              hooks.NewVMCryptoHook(),
+		InputParser:                             &mock.ArgumentParserMock{},
+		ValidatorAccountsDB:                     &stateMock.AccountsStub{},
+		ChanceComputer:                          &mock.RaterMock{},
+		EpochNotifier:                           &epochNotifier.EpochNotifierStub{},
+		SetSenderInEeiOutputTransferEnableEpoch: 0,
+	})
```

Comment:
can't we use `createDefaultEei()` here also?

Reply:
yes, fixed